docs/mounting.md

```diff
@@ -7,9 +7,28 @@
         gcloud auth login
         gcloud auth list
 
-   Alternatively, you can authenticate Cloud Storage FUSE by setting the ```--key-file``` flag to the path of a JSON key file, which you can download from the Google Cloud console. You can also set the ```GOOGLE_APPLICATION_CREDENTIALS``` environment variable to the path of the JSON key.
+   Alternatively, you can authenticate Cloud Storage FUSE using a service account
+key.
 
-        GOOGLE_APPLICATION_CREDENTIALS=/path/to/key.json gcsfuse [...]
+   - [Create a service account](https://cloud.google.com/iam/docs/service-accounts-create)
+  with one of the following roles:
+
+     * `Storage Object Viewer (roles/storage.objectViewer)` role to mount a
+    bucket with read-only permissions.
+     * `Storage Object Creator (roles/storage.objectCreator)`
+    and `Storage Object Viewer (roles/storage.objectViewer)` roles to
```

Comment:
does roles/storage.objectCreator allow deletion as well ? if not, can we look for a role (with minimum permissions) to do all the file system operations ?

Reply:
The [public documentation](https://cloud.google.com/storage/docs/access-control/iam-roles) description for Storage Object Creator role mentions that it allows deletion of objects which is incorrect. Updating this with Storage object Admin role. Thanks for pointing out.